README.md

```diff
@@ -1,9 +1,32 @@
-[![Stories in Ready](https://badge.waffle.io/BinaryBaboons/PlusOne.png?label=ready&title=Ready)](https://waffle.io/BinaryBaboons/PlusOne?utm_source=badge)
-# PlusOne
-PlusOne
+# EleventHour
 
+A social networking tool to fill the gaps in your event roster. Create an event and let people signup so that you don't have to cancel your rec-league game or board night.
```

Comment:
change this to read "rec-league game or board game night"?

Or maybe "rec-league or board game night."